events/viewsv2.py

```diff
@@ -2312,3 +2312,41 @@ def form_valid(self, form, **kwargs):
       print "saved"
       messages.success(self.request, "Thank you for completing this feedback form. We appreciate your input and valuable suggestions.")
       return HttpResponseRedirect(self.success_url)
+      
+  
+def verify_email(request):
+  template_name = 'verify_email.html'
+  context = {}
+  ci = RequestContext(request)
+  if request.method == 'POST':
+    email = request.POST.get('email').strip()
+    try:
+      user = User.objects.get(email = email)
+    except ObjectDoesNotExist:
+      context["notregistered"] = 1
+      return render_to_response(template_name, context, ci)
```

Comment:
You don't need a `RequestContext` if you can simply use `render` function do the job.

```
from django.shortcuts import render


def some_view(request, template_name='some/template.html'):
    # handle thing here
    context = {"foo": "bar"}
    return render(request, template_name, context)
````